src/test/java/com/aws/greengrass/logmanager/CloudWatchAttemptLogsProcessorTest.java

```diff
@@ -95,8 +94,9 @@ private void mockDefaultGetGroups() throws ServiceLoadException {
     }
 
     @Test
-    void GIVEN_one_system_component_one_file_less_than_max_WHEN_merge_THEN_reads_entire_file()
+    void GIVEN_one_system_component_one_file_less_than_max_WHEN_merge_THEN_reads_entire_file(ExtensionContext ec)
             throws URISyntaxException, ServiceLoadException {
+        ignoreExceptionOfType(ec, DateTimeParseException.class);
```

Comment:
Why do we need to ignore this exception?

Reply:
the exception's logged here https://github.com/aws-greengrass/aws-greengrass-log-manager/blob/14c9c769f74e0dd934f54522bd75c926fcfe0ce0/src/main/java/com/aws/greengrass/logmanager/CloudWatchAttemptLogsProcessor.java#L259-L262

Reply:
Ok. I was just confused that this should have been caught when that change was made.